client/src/containers/App/index.jsx

```diff
@@ -27,42 +30,72 @@ export class App extends React.Component {
   }
 
   render() {
-    const { app, history } = this.props;
+    const {
+      app, history, router, user,
+    } = this.props;
+
     return (
-      <IntlProvider id="locale-intl">
-        <LocaleProvider id="locale-antd" locale={app.locale.antd}>
-          <ConnectedRouter id="router" history={history}>
-            <Spin id="loading-animation" size="large" spinning={app.showLoadingAnimation}>
-              <div id="container">
-                <Navigation id="navigator" />
-                <Switch id="switch">
-                  <Route id="route-home" exact path="/" component={Home} />
-                  <Route id="route-example" exact path="/example" component={Example} />
-                  <Route id="route-nomatch" component={NoMatch} />
-                </Switch>
-              </div>
-            </Spin>
-          </ConnectedRouter>
+      <IntlProvider key="locale-intl">
+        <LocaleProvider key="locale-antd" locale={app.locale.antd}>
+          <TransitionGroup>
+            <CSSTransition
+              key={router.location.key}
+              timeout={{
+                enter: 300,
+                exit: 900,
+              }}
+              classNames={{
+                enter: 'animated fadeIn',
+                exit: 'animated fadeOut',
+              }}
+              mountOnEnter
+              unmountOnExit
+            >
+              <Spin id="loading-animation" size="large" spinning={app.showLoadingAnimation}>
+                <Layout id="layout">
```

Comment:
![image](https://user-images.githubusercontent.com/965232/54038973-ce5f6d80-418f-11e9-8d92-51c40049196a.png)

Reply:
@latch2112 I do not see this in my IDE. Can you show me please?